.gitignore

```diff
@@ -1 +1,5 @@
-/phpunit.xml
+phpunit.xml
+.DS_Store
+composer.lock
```

Comment:
See this: https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file